<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formulas_gui</class>
 <widget class="QWidget" name="Formulas_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../mathgraphica.qrc">
    <normaloff>:/images/images/formulas2.png</normaloff>:/images/images/formulas2.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>1) Formulas</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_new">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_del">
        <property name="text">
         <string>Del</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTableWidget" name="tableWidget_formulas">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <column>
         <property name="text">
          <string>Formulas</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Description</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit_min"/>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="lineEdit_max"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>min:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLineEdit" name="lineEdit_delta"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>delta:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>precision:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>max:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLineEdit" name="lineEdit_precision"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>3) Solutions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QListWidget" name="listWidget_solutions"/>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_solve">
        <property name="text">
         <string>Solve</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>2) Validate formula</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QTableWidget" name="tableWidget_constants">
        <column>
         <property name="text">
          <string>Constants</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Values</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_validate">
        <property name="text">
         <string>Validate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton_close">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../mathgraphica.qrc"/>
 </resources>
 <connections/>
</ui>
